src/imageproxy.rs

```diff
@@ -124,6 +124,10 @@ pub struct ImageProxyConfig {
     /// Defaults to `false`; in other words, use the default file paths from `man containers-auth.json`.
     pub auth_anonymous: bool,
 
+    // Directory with certificates (*.crt, *.cert, *.key) used to connect to registry
+    // Equivalent to `skopeo --src-cert-dir`
+    pub certificate_authority: Option<String>,
```

Comment:
Yeah let's rename this to `certificate_directory` and also make it `Option<PathBuf>` or so to make it explicitly clear it's a filesystem path.

Reply:
Should I change  `authfile` to `PathBuf` too?

Reply:
Hmm, yes.  In theory.  In practice, that's a semver-breaking change, so...hmm, yeah we can just bump semver for the next release I guess.